src/java/picard/analysis/directed/TargetMetricsCollector.java

```diff
@@ -653,6 +663,33 @@ public void finish() {
             return coverageDistribution;
         }
 
+        /** Emits a file with per base coverage if an output file has been set. */
+        private void emitPerBaseCoverageIfRequested() {
+            if (this.perBaseOutput == null) return;
```

Comment:
Yeah it's not a big deal, in fine with with ifRequested but since I'm supposed to be airing every petty grievance there ya go :)